common/src/Renderer/Camera.cpp

```diff
@@ -265,12 +265,14 @@ namespace TrenchBroom {
             if (direction == m_direction && up == m_up)
                 return;
             m_direction = direction;
-            if (m_direction.absolute().equals(up.absolute())) {
-                m_up = crossed(m_right, m_direction);
-            } else {
+            
+            // I can't remember what this is for, but this condition is too weak.
+            // if (m_direction.absolute().equals(up.absolute())) {
+            //    m_up = crossed(m_right, m_direction);
```

Comment:
I'll leave it in for now, maybe it turns out it was necessary, but the condition needs to be stronger.